docs/conf.py

```diff
@@ -47,7 +47,7 @@
 
 # General information about the project.
 project = u"pytest-selenium"
-copyright = u"2015, Dave Hunt"
+copyright = u"2019, Dave Hunt"
 author = u"Dave Hunt"
```

Comment:
I think we should add other authors here now.. my last commit was almost a year ago.